azurerm/internal/services/cosmos/tests/cosmosdb_account_resource_test.go

```diff
@@ -631,6 +631,7 @@ resource "azurerm_cosmosdb_account" "test" {
   geo_location {
     location          = "%[5]s"
     failover_priority = 1
+    is_zone_redundant = true
```

Comment:
rather than modifying a new test, can we add a new one specifically for this

Reply:
New test added